app/qmlV2/component/MMLinkButton.qml

```diff
@@ -0,0 +1,39 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Controls.Basic
+import "../Style.js" as Style
+import "."
+
+Button {
+  id: control
+
+  contentItem: Text {
+    anchors.centerIn: control
+    font: Qt.font(Style.t3)
+    text: control.text
+    leftPadding: 32 * __dp
```

Comment:
is this `*__dp` all around still needed in Qt 6.5.2?

Reply:
https://github.com/MerginMaps/input/commit/0022b537aa9ea5afa9232f5b6f668059d90be1c2